### PR TITLE
Add Furnace Model and Serial Number

### PR DESCRIPTION
**Pull request by stockmopar**

### CODE REVIEW

File: econet_hvac_furnace.yaml [hunk 1]
```diff
@@ -23,6 +25,21 @@ econet:
             id(static_pressure).publish_state(((x[15] << 8) + x[16]) / 5280.0);
             id(return_air_temperature).publish_state(((x[50] << 8) + x[51]) / 10.0);
             id(flame_sensor).publish_state(x[33] / 10.0);
+    - sensor_datapoint: HRHEEMID
+      request_mod: 5
+      src_address: 448
+      datapoint_type: raw
+      then:
+        - lambda: |-
+            if(x.size() > 52) {
```
Comment (tronikos) [inline econet_hvac_furnace.yaml]: Same as in the other PR. Space between if and parenthesis and negate to early return

File: econet_hvac_furnace.yaml [hunk 2]
```diff
@@ -23,6 +25,21 @@ econet:
...
+                std::string str(x.begin(), x.end());
+                std::string model = str.substr(4,24);
```
Comment (tronikos) [inline econet_hvac_furnace.yaml]: nit: space after comma

File: econet_hvac_furnace.yaml [hunk 3]
```diff
@@ -23,6 +25,21 @@ econet:
...
+                model.erase(remove(model.begin(), model.end(), '\00'), model.end());
+                id(hw_model_num).publish_state(model);
+                std::string serial = str.substr(28,24);
```
Comment (tronikos) [inline econet_hvac_furnace.yaml]: ditto

**Review by tronikos — CHANGES_REQUESTED** [review]

File: econet_hvac_furnace.yaml (hunk 1 above)
Comment (stockmopar) [reply econet_hvac_furnace.yaml]: Updated.

File: econet_hvac_furnace.yaml (hunk 3 above)
Comment (stockmopar) [reply econet_hvac_furnace.yaml]: Updated.

File: econet_hvac_furnace.yaml (hunk 2 above)
Comment (stockmopar) [reply econet_hvac_furnace.yaml]: Updated.

**Review by tronikos — APPROVED** [review]

**Pull request merged**